ibm/mas_devops/roles/suite_db2_setup_for_manage/templates/setupdb.sh.j2

```diff
@@ -64,13 +86,24 @@ db2set DB2_FMP_COMM_HEAPSZ=65536
 db2set DB2_SKIPDELETED=ON
 db2set DB2_USE_ALTERNATE_PAGE_CLEANING=ON
 
+echo "###  set the tmp backup location   ###"
+db2set DB2_OBJECT_STORAGE_LOCAL_STAGING_PATH=/mnt/backup/staging       ### Added
```

Comment:
As previous, no need to put `### Added` comments on all new lines introduced.

Reply:
Removed comment `### Added`